geth-with-protocol/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# can't use geth 1.8 - it doesn't retain state inside docker container
+FROM ethereum/client-go:v1.7.3 as builder-geth
+
+WORKDIR /
+ENV gethRoot /root/.ethereum
+COPY genesis.json /geth/genesis.json
+COPY keys/* /geth/keys/
```

Comment:
If we are using these keys, we should make sure people understand these are test keys.  I actually prefer to treat keys as disposable data, so we are always generating new ones (instead of checking these keys into the codebase).

Reply:
we need only first one, that is used for mining, all other not used, can be removed

Reply:
I think even the one address should be generated at start time.  We can share this key by adding a simple script, or using a shared volume.

Reply:
Is there any way to `Generate new “root” address that contains all the Eth` after blockchain was initialized?